__tests__/app.test.js

```diff
@@ -15,11 +13,10 @@ describe("API: /api/categories", () => {
       return request(app)
         .get("/api/categories")
         .expect(200)
-        .then(({ _body }) => {
-          console.log(_body);
-          expect(_body).toHaveLength(4);
-          expect(_body).toBeInstanceOf(Array);
-          _body.forEach((category) => {
+        .then(({ _body: { categories } }) => {
```

Comment:
You dont need `_body`. The key is `body`- no underscore, its not hidden or private.